README.md

```diff
@@ -311,6 +311,31 @@ sudo ln -s $(which rustc)
 
 This step needs to be executed each time Xcode updates.
 
+#### Building using Conda on MacOS
+
+We provide a Conda env for easy Setup. 
+Prequisits: 
+- Have miniconda installed. 
+
+```bash 
+$ conda env create -f env.yml
+$ conda activate VPN
+```
+ - Set a MacOS-SDK (via xcode, or the interwebs) 
+    - set `export SDKROOT=<something>`
+    - It's probably in:
```

Comment:
How does one find the path to the SDK root? Adding instructions to that would be more valuable than adding possible paths.

Reply:
rewrote that to make it more clear